assets/css/mawiblah.css

```
@@ -178,159 +191,155 @@ body {
+    overflow: hidden;
+    height: 40px;
 }
+
 .mawiblah-section {
-    position:relative;
-    width:100%;
+    position: relative;
+    width: 100%;
     margin-inline-end: 30px;
-    backgroud-color:red;
+    backgroud-color: red;
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)